subworkflows/local/fusioninspector_workflow.nf

```diff
@@ -42,18 +42,26 @@ workflow FUSIONINSPECTOR_WORKFLOW {
         FUSIONINSPECTOR( ch_reads_fusion, ch_starfusion_ref)
         ch_versions = ch_versions.mix(FUSIONINSPECTOR.out.versions)
 
-        if(!skip_vcf) {
-            AGAT_CONVERTSPGFF2TSV(FUSIONINSPECTOR.out.out_gtf)
+        def tsv_nonempty = FUSIONINSPECTOR.out.tsv.filter { _meta, file -> file.exists() && file.size() > 0 }
+        def tsv_abridged_nonempty = FUSIONINSPECTOR.out.abridged_tsv.filter { _meta, file -> file.exists() && file.size() > 0 }
+
+        def gtf_nonempty = FUSIONINSPECTOR.out.out_gtf.filter { _meta, file -> file.exists() && file.size() > 0 }
```

Comment:
Are these files fully empty or does the header remain when there are no fusions?

Reply:
they should be empty